docs/proposals/modifying-output-image.md

```diff
@@ -0,0 +1,126 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: modifying-output-image
+authors:
+  - "@ImJasonH"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-04-14
+last-updated: 2021-04-14
+status: provisional
+---
+
+# Modifying Output Image
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions
+
+1. Should this feature also allow users to _unset_ existing labels/annotations inherited from base images?
```

Comment:
Does not sound too relevant to me. Is it possible to just assume that an entry with an empty string value means that the user wants to unset it? Or is an empty string a possible and common value?

Reply:
Unfortunately it's also possible to have labels/annotations with empty values, and some user might theoretically handle those differently.

I do agree that this probably isn't a necessary use case to solve immediately, we can wait for some user to ask for it and use that as an opportunity to ask how they'd like to surface that feature.